README.md

```diff
@@ -73,3 +73,28 @@ The architecture is as follows:
 - Each destination is expected to be a PDA that then distributes the yield to the climate project
   according to their own rules. For example, the Offset Bridge bridges the funds to Polygon, buys
   NCT etc.
+
+## Usage (latest version -- Yield Router)
+There is a state account owned by the yield controller program which stores the following data:
+- `sunrise_state`: public key of the input yield account which holds funds to be sent to PDAs that will transfer fund to various climate projects (e.g. the offset bridge)
```

Comment:
I think sunrise_state is not the public key of the input yield account but a pda that holds data about sunrise stake state.